variables.tf

```diff
@@ -321,3 +321,20 @@ variable "max_instance_lifetime" {
   default     = 0
 }
 
+variable "instance_refresh_minimum_health_percentage" {
+  description = "The amount of capacity in the Auto Scaling group that must remain healthy during an instance refresh to allow the operation to continue, as a percentage of the desired capacity of the Auto Scaling group."
+  default     = 50
```

Comment:
Why `50`? [Magic number](https://en.wikipedia.org/wiki/Magic_number_(programming))?

Reply:
Make it default to `null` to use the provider's default which is `90`. Ref: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/autoscaling_group#min_healthy_percentage

Reply:
done! @spatel96 can u set as resolved?